test/integration/helpers.js

```diff
@@ -519,21 +520,21 @@ function replaceFileContents(filepath, pattern, replacement) {
  */
 function copyFixture(fixtureName, dest) {
   const fixtureSource = resolveFixturePath(fixtureName);
-  fs.ensureDirSync(path.dirname(dest));
-  fs.copySync(fixtureSource, dest);
+  fs.mkdirSync(path.dirname(dest), { recursive: true });
+  fs.cpSync(fixtureSource, dest);
```

Comment:
From the CI logs, it seems this API is (the only one) missing from Node.js v14. It looks like that version of Node.js only has functions to copy individual files.

@JoshuaKGoldberg I have three suggestions for moving forward:

1. Write a custom implementation of `fs.cpSync` here.
2. Postpone this change until Node.js v14 is no longer run in CI.
3. Drop the Node.js v14 job since it's no longer [supported](https://github.com/mochajs/mocha/blob/664e1f49f7ae214a9666c90f388407e9fa100309/package.json#L43).

Reply:
Fun timing: #5279 just dropped v14 in CI. If you update this branch from `main` it should be good to go! 🚀

Reply:
Oh, it _is_ updated from `main`. Investigating...